engine/ai.py

```diff
@@ -42,7 +42,7 @@ class AI:
     def __init__(self):
         self.owner = None  # Fully mutable member.
 
-    def choose_action(self, room, impulse=None):
+    def choose_action(self, unused_room, impulse=None):
```

Comment:
Why not "unused_impulse," too?

Reply:
Because I'm just doing this so that the method signatures match :)